cli/js/web/url_encoding.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
```

Comment:
Does this have an equivalent in the DOM spec, even internal structures.  Pretty much every unique module we have in `js/web` corresponds to something in the DOM spec.  If URL is only using it, does it need to be seperate?

Reply:
Moved. Thought it'd be longer and noisier and at first.